send_email.py

```diff
@@ -124,8 +128,13 @@ def main():
             length,
             camera_id_id
         )
-        times = select_all_times(connection, file_name)
-        send_email(times, file_path, file_name)
+        times = select_all_times(connection)
+        send_email(
```

Comment:
What is sent if the file doesn't exist? ie, if record to device is turned off, should it still try to send the file? 

I think we want to check if the file exists, and if not, we only want to send the database record and not the attachment?